src/app/submission/sections/container/section-container.component.ts

```diff
@@ -0,0 +1,93 @@
+import { Component, Injector, Input, OnInit, ViewChild } from '@angular/core';
+
+import { SectionsDirective } from '../sections.directive';
+import { SectionDataObject } from '../models/section-data.model';
+import { rendersSectionType } from '../sections-decorator';
+import { AlertType } from '../../../shared/alert/aletr-type';
+
+/**
+ * This component represents a section that contains the submission license form.
+ */
+@Component({
+  selector: 'ds-submission-section-container',
```

Comment:
This selector name isn't coherent with others.

Reply:
I think that this follows the same rule used for other sections components